src/desktop/apps/order2/server.js

```diff
@@ -6,9 +6,14 @@ import { buildServerApp } from "reaction/Artsy/Router/server"
 import styled from "styled-components"
 import { buildServerAppContext } from "desktop/lib/buildServerAppContext"
 
-const app = (module.exports = express())
+const app = express()
 
-app.get("/order2/:orderID*", async (req, res, next) => {
+export const appHandler = async (req, res, next) => {
```

Comment:
We tend to name routes in a more specific/explicit way. I think this would be clearer if it were called `orderApp` or `checkoutFlow` or something like that.